adapters/repos/db/crud_integration_test.go

```diff
@@ -1853,3 +2001,13 @@ func randomVector(dim int) []float32 {
 
 	return out
 }
+
+func newBeacon(className string, id strfmt.UUID) strfmt.URI {
+	return strfmt.URI(
+		fmt.Sprintf("weaviate://localhost/%s/%s",
+			className, id))
```

Comment:
I know it's test but there's a dedicated package (`crossref`) for managing beacons, there's even a method there that automatically creates `SingleRef` objects

Reply:
Thanks